airtable/airtable_to_mailing_list_test.py

```diff
@@ -0,0 +1,81 @@
+"""Test airtable_to_mailing_list.py"""
```

Comment:
I think `test_file_name.py` is more standard than `file_name_test.py`, but I can't find a source easily, so up to you.